docs/downstream-integrations/api/authentication.md

```diff
@@ -24,30 +24,130 @@ The AE token must be passed in the HTTP `Authorization` (case insensitive) reque
+- **Client Secret**: A confidential string used to authenticate your client application with Collab.Land.
+- **API Key**: A secret key used used as a convenient way to invoke Collab.Land APIs that don't involve any users, such as `token-gating` services.
 
+These credentials are essential for initiating the OAuth2 flow and obtaining access tokens.
 
-The `redirect_uri` parameter is required. This is the URL that the user will be redirected to after the authentication process is complete. 
+## Understand Scopes
 
-> The `LWC` flow is not publicly available yet, so it only works in development and with whitelisted domains. If you want to use this method, be sure to inform the Collab.Land team before development or submit a request via [the API Request Form](https://forms.gle/GbtyiQyBkUH1bwsL8)
+Collab.Land API uses scopes to define the level of access and permissions requested by the client application. Scopes determine the resources and operations that the access token can be used for. It is crucial to understand the available scopes and select the appropriate ones based on your application's requirements.
 
-<!-- ### Sign in with Discord or Telegram
+The following scopes are available:
 
-![Sign in with Discord 1](../imgs/login.png)
+**account:read**
 
-You should receive a confirmation page when you sign in successfully via your authentication platform of choice. For instance, an authenticated wallet connection confirmation page would look like this:
+- Read all the wallet address(es) you connected with Collab.Land and their balances.
+- Read basic user information including username, id , pfp.
+- See communities you are in and administrating.
 
-![Sign in with Discord 2](../imgs/connected.png) -->
+**access-control:read**
+Read user roles in a certain community
+
+**asset-ownership:read**
+Check ownership for the given account for the given token assets
+
+**gm-pass:read**
+Read gmPASS address and DID info
+
+**read-vcs**
+Read all public verifiable credentials issued to you
+
+**request-vcreds**
+Issue verifiable credentials to your DID (gmPASS)
+
+**kudos:read**
+Read certain type public verifiable credentials issued to you
+
+:::success Please be aware that these scopes are currently being revised and will soon be updated to ensure accurate naming conventions and consistency.
+:::
+
+## Login with Collab.Land (LWC) Flow
+
+To initiate the OAuth2 flow, direct users to the Collab.Land OAuth2 authorization endpoint:
+
+```bash
+https://api.collab.land/oauth2/authorize
+```
+
+Include the following parameters in the URL:
+
+- **`response_type`**: Set this to **`code`** for the authorization code flow or **`token`** for the implicit token flow.
+- **`client_id`**: The client ID obtained during the application registration process.
+- **`redirect_uri`**: The URI to which Collab.Land will redirect the user after authentication. Must be one of your registered redirect URIs. Register your localhost URIs for testing.
```

Comment:
I improved the redirect uri mapping for local dev, we allow:
- localhost
- 127.0.0.1
- [::1] (ipv6)
The port number does not matter and subdirs are fine.

For example, if we register `http://localhost:3000/oauth2` on the dev portal as one of the redirect uris, we allow the following:
http://localhost:5000/oauth2/implicit-flow.html
http://127.0.0.1:3001/oauth2/authorization-code-flow.html